configs/server/forseti_conf_server.yaml.in

```diff
@@ -62,8 +62,8 @@ inventory:
           period: 1.1
 
     cai:
-        enabled: False
-        gcs_path: ""
+        enabled: True
```

Comment:
Should we make this a variable as well? So it can be set false if the root is not an organization?